ansible_collections/arista/avd/roles/dhcp_provisioner/templates/ztp_configuration.j2

```diff
@@ -1,4 +1,10 @@
 {# HEADER -- LOGIC implementation #}
+{% set cvp_instance_ip = UNSET %}
```

Comment:
with this I got 
```
TASK [arista.avd.dhcp_provisioner : Build DHCP intended variables] ***********************************************************
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ansible.errors.AnsibleUndefinedVariable: 'UNSET' is undefined
fatal: [AVD-CVP-1 -> localhost]: FAILED! => {"changed": false, "msg": "AnsibleUndefinedVariable: 'UNSET' is undefined"}

NO MORE HOSTS LEFT ***********************************************************************************************************

PLAY RECAP *******************************************************************************************************************
AVD-CVP-1                  : ok=1    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0
```

and when putting it in quotes the playbook worked however the `ztp_configuration.yml` has put that into the `registration` url
```
---
ztp:
  default:
    registration: http://UNSET/ztp/bootstrap
```

I guess that's because I do not use eos_designs to generate the TA config?

Reply:
Yes it is expected as ansible error message is not clear, template now generate everything and put an error message in the configuration file that any user should review before pushing to production.

Message has been updated with the following message: `cvp_instance_ip is unset - please set`